Makefile

```diff
@@ -107,9 +107,6 @@ webots_target: webots_dependencies
 	@+make --silent -C src/ode $(TARGET)
 ifeq ($(TARGET),profile)  # a shared version of the library is required for physics-plugins
 	@+make --silent -C src/ode release
-endif
-ifneq ($(TARGET),clean)
-	@+make --silent -C src/ode install
```

Comment:
I tested locally and libode is no longer copied in the `lib/webots` folder after compiling.
I'm inspecting why it doesn't work locally but worked on the CI.
Maybe it is just a local issue.

Reply:
Yes, I simplified it as the `make install` target seemed over-complicated and superfluous to me.